resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "dateOfBirth": "1985-03-25"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">201 (Created)</code></p>
+        <pre class="code--block">
+        {
+          "status": 201,
+          "success": true,
+          "data": {
+            "message": "Investor created",
+            "investorId": "9876543210"
+          }
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request with a valid payload and an invalid LISA Manager reference number</p> <p class="code--block">lisaManagerReferenceNumber: 123456</p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorNINO": "AA123456A",
+          "firstName": "First Name",
+          "lastName": "Last Name",
+          "dateOfBirth": "1985-03-25"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Enter lisaManagerReferenceNumber in the correct format, like Z1234"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing invalid and/or missing data</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorNINO": "A1234567A",
+          "firstName": true,
+          "dateOfBirth": "25-03-1985"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Bad Request",
+          "errors": [
+            {
+              "code": "MISSING_FIELD",
+              "message": "This field is required",
+              "path": "/lastName"
+            },
+            {
+              "code": "INVALID_DATE",
+              "message": "Date is invalid",
+              "path": "/dateOfBirth"
+            },
+            {
+              "code": "INVALID_FORMAT",
+              "message": "Invalid format has been used",
+              "path": "/investorNINO"
+            },
+            {
+              "code": "INVALID_DATA_TYPE",
+              "message": "Invalid data type has been used",
+              "path": "/firstName"
+            }
+          ]
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing investor details which do not match HMRC’s records</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks